.gitignore

```diff
@@ -6,3 +6,5 @@
 /.gwt
 /build
 /.gradle
+war/
+.settings/
```

Comment:
I generally check in .settings files so that contributors get the formatter/compiler/etc. settings setup automatically.